app/views/availabilities/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <b>Person:</b>
+  <%= link_to @availability.person.fullname, @availability.person %>
+</p>
+
+<% if @availability.event.present? %>
+  <p><b>Event:</b><%= link_to @availability.event.title, @availability.event %></p>
+<% end %>
+
+<p>
+  <strong>Start time:</strong>
+  <%= @availability.start_time %>
+</p>
+
+<p>
+  <strong>End time:</strong>
+  <%= @availability.end_time %>
```

Comment:
That is a good idea, and may be handled partially by the calendar library. I will create an issue, as I don't want to hold up this pr for that.